frontend/src/toolbar/posthog.ts

```diff
@@ -4,10 +4,12 @@ const DEFAULT_API_KEY = 'sTMFPsFhdP1Ssg'
 
 const runningOnPosthog = !!window.POSTHOG_APP_CONTEXT
 const apiKey = runningOnPosthog ? window.JS_POSTHOG_API_KEY : DEFAULT_API_KEY
-const apiHost = runningOnPosthog ? window.JS_POSTHOG_HOST : 'https://app.posthog.com'
+const apiHost = runningOnPosthog ? window.JS_POSTHOG_HOST : 'https://d1iyf54ud053a4.cloudfront.net'
+const uiHost = runningOnPosthog ? window.JS_POSTHOG_UI_HOST : 'https://app.posthog.com'
```

Comment:
This is onlyused by things like the sentry integration in order to have urls that link to the web app. It isn't necessary for js-lite.